values.schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "airflow": {
+      "type": "object"
+    },
+    "authSidecar": {
+      "type": "object"
+    },
+    "extraObjects": {
+      "type": "array"
+    },
+    "helmValuesRevisionId": {
+      "type": "integer"
+    },
+    "ingress": {
+      "type": "object"
+    },
+    "platform": {
+      "type": "object"
+    },
+    "sccEnabled": {
+      "type": "boolean"
+    },
+    "workers": {
+      "type": "object"
+    }
```

Comment:
It might be worth expanding this to cover everything we currently support, as `workers.extraVolumeMounts` for example would pass validation but no longer work.

Reply:
I would say that if `workers.extraVolumeMounts` used to work and after the OSS helm chart switch it does not work, it's worth doing. @jedcunningham do you know if that is the case?

Reply:
I'll update the schema file to include declaration for all top-level properties (and their child properties) except for `airflow` which is delegated to the OSS chart. Would that work? Any other concerns?

Reply:
In this first pass, only `workers` needs to be comprehensive. The others didn't get moved (and in the case of workers, partially moved).

Reply:
Thanks, @jedcunningham. I added types/properties for everything under `workers`.